api/lib/infrastructure/repositories/training-trigger-repository.js

```diff
@@ -70,7 +71,19 @@ module.exports = {
 };
 
 async function _toDomain({ trainingTrigger, triggerTubes, tubes = [] }) {
-  const learningContent = await _getLearningContent(triggerTubes);
+  const notFoundTubeIds = triggerTubes.filter(({ tubeId }) => {
+    return !tubes.map((tube) => tube.id).includes(tubeId);
```

Comment:
C'est un peu dommage de recalculer le tableau d'id des sujets à chaque itération. 
On pourrait le remonter